spec/openai/client/http_spec.rb

```diff
@@ -134,44 +139,52 @@
         it "does not call the user proc" do
           bad_examples.each do |chunk|
             expect(user_proc).to_not receive(:call)
-            stream.call(chunk)
+            stream.call("#{chunk}\n\n")
           end
         end
       end
 
       context "when called with a string containing that looks like a JSON object but is invalid" do
-        let(:chunk) do
-          <<-CHUNK
-            data: { "foo": "bar" }
-            data: { BAD ]:-> JSON }
```

Comment:
So, this was actually a totally new case that doesn't have to be handled in the scope of OpenAI responses because they don't seem to return anything like this. Normally, adjacent `data` fields that the values should be concatenated with a `\n` -- it's a multi-line value. In this case, the above would be read as `"{ "foo": "bar" }\n{ BAD ]:-> JSON }"` and that is not what this test's intention is.